### ISSUE TITLE
CSRNG usually spelled CSPRNG

### ISSUE THREAD

**Issue by cemeyer** [RustCrypto/CSRNGs#4]
This is a minor nit, but given there is no code here yet, is renaming to the more conventional spelling an option?

**Comment by newpavlov** [comment]
Yes, it's an option, though CSRNG was selected intentionally, as crates in this repository may cover hardware RNGs in future (e.g. something like `rdrand` crate).
